src/main/scala/com/metl/model/BackendAdaptor.scala

```diff
@@ -550,3 +551,28 @@ class ResourceCachingAdaptor(sc:ServerConfiguration,cacheConfig:CacheConfig) ext
     super.isReady
   }
 }
+
+class D2LCachingAdaptor(cacheConfig:CacheConfig) {
```

Comment:
my intention was to cache the things that take a long time, and that's the d2l query.  i want the report content to change in real time.  currently this report takes about 1s to run the second time around (after d2l is cached).  it's not much slower in prod over Maridelys' course